### PR TITLE
[TACHYON-1648]Add the Chinese version of documentation page 'Hadoop MapReduce'

### PR DESCRIPTION
**Pull request by summerDG**
Add the Chinese version of documentation page 'Hadoop MapReduce'. See https://tachyon.atlassian.net/browse/TACHYON-1648

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by ifcharming** [comment]
LGTM

Thank you for the translation!

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+layout: global
+title: 在Tachyon上运行Hadoop MapReduce
+nickname: Apache Hadoop MapReduce
+group: Frameworks
+priority: 1
+---
+
+该文档介绍如何让Tachyon与Apache Hadoop MapReduce一起运行，从而让你轻松地运行文件存储在Tachyon上的MapReduce项目。
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: ”项目“ ->”程序“

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+如果运行的是Hadoop 1.x集群，确保在Hadoop的安装目录下的`conf`目录中有`core-site.xml`文件，并且已经加入了如下属性：
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/config-core-site.md %}
+
+该配置让你的MapReduce作业可以使用Tachyon来输入输出文件。如果你正在使用HDFS作为Tachyon的底层存储系统，同样有必要在`hdfs-site.xml`文件中添加这些属性：
+
+## 使用Hadoop 2.x
+
+如果你正在使用2.x版本的Hadoop集群，你应该不需要像上面那样在`core-site.xml`文件中添加属性。可是，有些情况下可能会遇到`java.io.IOException: No FileSystem for scheme: tachyon`的错误。例如，当Yarn(与Hadoop相对)尝试去访问Tachyon文件时，可能发生该错误。如果遇到该错误，在`core-site.xml`文件中添加这些属性，然后重启Yarn。
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: Yarn -> YARN

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/config-core-site.md %}
+
+# 编译Tachyon客户端
+
+为了使Tachyon和你的Hadoop版本相适应，你必须重新编译Tachyon Client的Jar包，指明你的Hadoop版本。你可以在Tachyon目录下运行如下命令：
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: 适应 -> 对应

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+{% include Running-Hadoop-MapReduce-on-Tachyon/compile-Tachyon-Hadoop.md %}
+
+`<YOUR_HADOOP_VERSION>`版本支持很多不同的Hadoop发行版。例如：`mvn install -Dhadoop.version=2.7.1`将会编译出适合Apache Hadoop 2.7.1版本的Tachyon。 
+请访问[构建Tachyon主分支](Building-Tachyon-Master-Branch.html#distro-support)页面来获取其他发行版本的支持信息。
+
+编译成功后，新的Tachyon客户端Jar包可以在如下目录中找到：
+
+    core/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar
+
+该Jar包会应用于下面的内容。
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: 这句改成”本页面文档后续的内容都是基于这个Jar而展开“。

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+为了可以使Tachyon客户端Jar包对JobClient生效，你可以在`hadoop-env.sh`文件中将`HADOOP_CLASSPATH`修改为：
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/config-hadoop.md %}
+
+该配置让代码可以使用Tachyon的URI来创建和提交作业。
+
+# 分发Tachyon客户端Jar包
+
+为了让MapRedude作业可以在Tachyon上读写文件，Tachyon客户端Jar包必须被分发到集群的所有节点上。从而使TaskTracker和JobClient有必要的可执行文件同Tachyon进行交互。
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: 这句”从而使TaskTracker和JobClient有必要的可执行文件同Tachyon进行交互。“改成”这使得TaskTracker和JobClient包含所有与Tachyon进行交互访问所需要的可执行程序“。

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+1.**使用-libjars命令行选项**
+你可以在使用`hadoop jar ...`的时候加入-libjars命令行选项，指定`/<PATH_TO_TACHYON>/core/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar`为参数。这条命令会把该Jar包放到Hadoop的DistributedCache中，使所有节点都可以访问到。例如，下面的命令就是将Tachyon客户端Jar包添加到`-libjars`选项中。
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/add-jar-libjars.md %}
+
+2.**手动将Jar包分发到所有节点**
+为了在每个节点上安装Tachyon，你必须将客户端Jar包`tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar`（位于`/<PATH_TO_TACHYON>/core/client/target/`目录）放到每个MapReduce节点的`$HADOOP_HOME/lib`（由于版本不同也可能是`$HADOOP_HOME/share/hadoop/common/lib`）目录下，然后重新启动所有的TaskTracker。该方法要注意的是所有Jar包必须再次安装，因为每个Jar包都更新到了最新版本。另一方面，当该Jar包已经在每个节点上的时候，就没有必要使用`-libjars`命令行选项了。
+
+# 本地运行使用Tachyon的Hadoop wordcount
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: 在本地模式的Tachyon上运行Hadoop wordcount

File: docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+为了方便，我们假设是伪分布式的集群，通过运行如下命令启动：
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/start-cluster.md %}
+
+配置Tachyon，将本地HDFS集群作为其底层存储系统。你需要修改`conf/tachyon-env.sh`，加入如下语句：
+
+{% include Running-Hadoop-MapReduce-on-Tachyon/config-Tachyon.md %}
+
+本地启动Tachyon：
```
Comment (RongGu) [inline docs/cn/Running-Hadoop-MapReduce-on-Tachyon.md]: 改成"以本地模式启动Tachyon"

**Comment by RongGu** [comment]
Thanks for this PR. I left some comments @summerDG . Please find some time to address that. Happy Spring Festival!

**Comment by RongGu** [comment]
BTW, please remember to add a screencast of the html page.

**Comment by summerDG** [comment]
http://www.tachyon-project.org/documentation/master/en/Running-Hadoop-MapReduce-on-Tachyon.html

**Comment by summerDG** [comment]
a
b
c
d
e

**Comment by summerDG** [comment]
a
b
c
d
f

**Comment by yupeng9** [comment]
LGTM. Thanks for the translation!

**Pull request merged**